### PR TITLE
migrate all urls to https

### PR DESCRIPTION
**Pull request by Zolmeister**

### CODE REVIEW

File: server.coffee [hunk 1]
```diff
@@ -229,9 +229,10 @@ renderHomePage = do ->
     icon440x280: 'https://cdn.wtf/d/images/icons/icon_440_280.png'
     # can't specify https because:
     # https://github.com/kikinteractive/kik-js-issues/issues/12
-    iconKik: '//cdn.wtf/d/images/icons/icon_256_orange.png'
-    url: 'http://clay.io/'
-    canonical: 'http://clay.io'
+    iconKik: 'https://cdn.wtf/d/images/icons/icon_256_orange.png'
```
Comment (austinhallock) [inline server.coffee]: rm comment

File: src/services/url.coffee [hunk 2]
```diff
@@ -11,12 +11,13 @@ class UrlService
     return host is config.HOST
 
   getMarketplaceBase: ({protocol} = {}) ->
-    protocol ?= if EnvironmentService.isKikEnabled() then 'card' else 'http'
+    protocol ?= if EnvironmentService.isKikEnabled() then 'card' else 'https'
```
Comment (austinhallock) [inline src/services/url.coffee]: 'cards' (+ anywhere else you see the card protocol)

File: src/services/url.coffee [hunk 3]
```diff
@@ -26,7 +27,7 @@ class UrlService
   # returns the full url to a game's subdomain page (eg http://slime.clay.io)
   getGameSubdomain: ({game, protocol}) =>
     protocol ?= if EnvironmentService.isKikEnabled() and @isRootPath() \
-                then 'card' else 'http'
+                then 'card' else 'https'
```
Comment (austinhallock) [inline src/services/url.coffee]: 'cards'

**Comment by Zolmeister** [comment]
updated

**Pull request merged**